python/ray/util/client/dataclient.py

```diff
@@ -35,6 +34,11 @@ def __init__(self, channel: "grpc._channel.Channel", client_id: str,
         self.ready_data: Dict[int, Any] = {}
         self.cv = threading.Condition()
```

Comment:
what is self.cv ? can we use another name for this parameter?

Reply:
It is a condition variable. This is used pretty consistently throughout the Ray Client codebase.